package com.proje.statement3;

import java.sql.SQLException;

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

import com.mysql.jdbc.Connection;
import com.mysql.jdbc.Statement;
import com.proje.statement2.DBConnection2;

public class DBProcesses {
	
	private final Logger logger = LogManager.getLogger();
	
	public void createPersonelTable(){
		
		
		Connection connection = DBConnection3.getConnection();
	
		Statement statement = null;
		
		try {
			statement = (Statement) connection.createStatement();
			
			String sorgu = "CREATE TABLE personel(personelId INT PRIMARY KEY NOT NULL, adi VARCHAR(255), soyadi VARCHAR(255), dogumYili INT,personelNumarasi VARCHAR(20))";
			
			statement.execute(sorgu);
			
			logger.info("Personel tablosu baþarýlý bir þekilde oluþturuldu.");
		} catch (SQLException e) {
			
			logger.warn("Tablo oluþturulurken hata meydana geldi.");
			return;
		}finally {
		
			DBConnection3.closeConnection(connection,statement);
			
		}
	}

}
